### DISCUSSION TITLE
Refactor training service, add non-running mode

### DISCUSSION THREAD

**Discussion by SparkSnail** [microsoft/nni#3771] [Ideas]
Refactor trainingService, give interface to users, let users call trainingService function directly from nnictl. For example, move delete experiment feature to trainingService, nnictl will call function from trainingService to delete experiment.

**Comment by QuanluZhang** [comment]
@SparkSnail can we close this issue now?

**Comment by SparkSnail** [comment]
I don't remember the scenario to use this feature, perhaps we could have a short discussion about this feature, and close it if it's not necessary to be implemented.

**Comment by scarlett2018** [comment]
> I don't remember the scenario to use this feature, perhaps we could have a short discussion about this feature, and close it if it's not necessary to be implemented.

any conclusion from the discussion?
